kill-halted-tasks/airflow-kill-halted-tasks.py

```diff
@@ -118,7 +119,7 @@ def kill_halted_tasks_function(**context):
         process = parse_process_linux_string(line=line)
 
         logging.info("Checking: " + str(process))
-        execution_date_to_search_for = str(process["airflow_execution_date"]).replace("T", " ") + "%"
+        execution_date_to_search_for = datetime.strptime((process["airflow_execution_date"]).replace("T", " "),'%Y-%m-%d %H:%M:%S.%f').replace(tzinfo=timezone('UTC'))
```

Comment:
We need to test this to ensure that it will find tasks and actually kill them

Reply:
Tested the changes and it is killing the process successfully.